### PR TITLE
display cache hit rate through jconsole

### PR DESCRIPTION
**Pull request by little-emotion**
1. View cache hit rate through debug level log and MXBean.
2. Log configuration file for iotdb are loaded dynamically every 60s

### CODE REVIEW

**Comment by LeiRui** [comment]
excellent!

File: server/src/assembly/resources/conf/logback.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     under the License.
 
 -->
-<configuration>
+<configuration scan="true" scanPeriod="100 seconds">
```
Comment (jixuan1989) [inline server/src/assembly/resources/conf/logback.xml]: 60s is fine. It is a good feature, you'd better point out this change in the commitlog.

File: server/src/main/java/org/apache/iotdb/db/service/ServiceType.java [hunk 2]
```diff
@@ -35,6 +35,7 @@
   PERFORMANCE_STATISTIC_SERVICE("PERFORMANCE_STATISTIC_SERVICE","PERFORMANCE_STATISTIC_SERVICE"),
   MANAGE_DYNAMIC_PARAMETERS_SERVICE("Manage Dynamic Parameters", "Manage Dynamic Parameters"),
   TVLIST_ALLOCATOR_SERVICE("TVList Allocator", ""),
+  CACHE_HIT_RATE_DISPLAY_SERVICE("CACHE_HIT_RATE_DISPLAY_SERVICE","Cache Hit Rate"),
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/service/ServiceType.java]: I'd like use the following method:
```
FLUSH_SERVICE("Flush ServerService",
      generateJmxName("org.apache.iotdb.db.engine.pool", "Flush Manager"));
```
Then you do not need to maintain the mbeanName in CacheHitRateMonitor.

File: server/src/assembly/resources/conf/logback.xml (hunk 1 above)
Comment (little-emotion) [reply server/src/assembly/resources/conf/logback.xml]: ok

File: server/src/main/java/org/apache/iotdb/db/service/ServiceType.java (hunk 2 above)
Comment (little-emotion) [reply server/src/main/java/org/apache/iotdb/db/service/ServiceType.java]: changed

File: server/src/main/java/org/apache/iotdb/db/engine/cache/DeviceMetaDataCache.java [hunk 3]
```diff
@@ -153,6 +147,29 @@ public static DeviceMetaDataCache getInstance() {
...
+    if (isHit) {
+      logger.debug(
+          "[ChunkMetaData cache hit] The number of requests for cache is {}, hit rate is {}.",
+          cacheRequestNum.get(), cacheHitNum.get() * 1.0 / cacheRequestNum.get());
+    } else {
+      logger.debug(
+          "[ChunkMetaData cache didn't hit] The number of requests for cache is {}, hit rate is {}.",
+          cacheRequestNum.get(), cacheHitNum.get() * 1.0 / cacheRequestNum.get());
+    }
+  }
```
Comment (fanhualta) [inline server/src/main/java/org/apache/iotdb/db/engine/cache/DeviceMetaDataCache.java]: These codes could be more concise.
```suggestion
      logger.debug(
          "[ChunkMetaData cache {}hit] The number of requests for cache is {}, hit rate is {}.",
          isHit ? "":"didn't ", cacheRequestNum.get(), cacheHitNum.get() * 1.0 / cacheRequestNum.get());
```

File: server/src/assembly/resources/conf/logback.xml (hunk 1 above)
Comment (jt2594838) [reply server/src/assembly/resources/conf/logback.xml]: Do not forget to update the related descriptions in docs.

**Pull request merged**